consensus/service/config/src/lib.rs

```diff
@@ -125,6 +125,11 @@ pub struct Config {
     /// The configured block version
     #[clap(long, default_value = "0", value_parser = parse_block_version, env = "MC_BLOCK_VERSION")]
     pub block_version: BlockVersion,
+
+    /// Maximum number of client session tracking structures to retain in
+    /// a least-recently-used cache.
+    #[clap(long, default_value = "4096", env = "MC_CLIENT_TRACKING_CAPACITY")]
```

Comment:
How was `4096` value derived?

Reply:
As discussed in standup - thanks for calling this out, asking about it means James was able to point me towards a similar LRU in the attest code. 
This should be resolved in https://github.com/mobilecoinfoundation/mobilecoin/pull/3155/commits/e67a060c4739e876863dbf1d781240b0e665cdb4

Reply:
Thanks Millie for clarifying this :)